modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -1082,9 +1101,9 @@ public void testCreateZone() {
 
         // Validate catalog version from the past.
         assertNull(manager.zone(zoneName, 0));
-        assertNull(manager.zone(2, 0));
+        assertNull(manager.zone(3, 0));
```

Comment:
Here, you can change check order to avoid guessing zone id, and get smth like
```suggestion
        assertNull(manager.zone(zone.id(), 0));
```

Reply:
The comment above is a bit misleading - the code checks that if we give a catalog manager id of not existing zone it is going to return null. Tweak this code a little bit and added a comment.